content/docs/strict-mode.md

```diff
@@ -1,65 +1,65 @@
 ---
 id: strict-mode
-title: Strict Mode
+title: Modo Estrito
 permalink: docs/strict-mode.html
 ---
 
-`StrictMode` is a tool for highlighting potential problems in an application. Like `Fragment`, `StrictMode` does not render any visible UI. It activates additional checks and warnings for its descendants.
+O modo estrito (`Strict Mode`) é uma ferramenta para sinalizar potenciais problemas em uma aplicação. Assim como o `Fragment`, o `StrictMode` não renderiza nenhum elemento visível na interface. Ele ativa, no entanto, verificações e avisos adicionais para os seus componentes filhos.
 
-> Note:
+> Nota:
 >
-> Strict mode checks are run in development mode only; _they do not impact the production build_.
+> Verificações do modo estrito são executadas somente em modo de desenvolvimento; _elas não impactam na build de produção_.
 
-You can enable strict mode for any part of your application. For example:
+Você pode habilitar o modo estrito para qualquer parte da sua aplicação, por exemplo:
 `embed:strict-mode/enabling-strict-mode.js`
 
-In the above example, strict mode checks will *not* be run against the `Header` and `Footer` components. However, `ComponentOne` and `ComponentTwo`, as well as all of their descendants, will have the checks.
+No exemplo acima, as verificações do modo estrito não serão executadas nos componentes `Header` e `Footer`. No entanto, `ComponentOne` e `ComponentTwo`, assim como todos os seus componentes filhos, serão verificados.
```

Comment:
see comment above re "filhos"

Reply:
In this case, instead of `para os componentes que descendem dele`, I used `[...] seus componentes descendentes [...]`.